docs/content/annotations.md

```diff
@@ -50,149 +50,35 @@ a metadata block determines how that metadata block will be applied. If the
 immediately follows the annotation. The `scope` values that are currently
 supported are:
 
-* `rule` - applies to the individual rule statement. Default, when metadata block precedes rule.
-* `document` - applies to all of the rules with the same name in the same package
-* `package` - applies to all of the rules in the package, Default, when metadata block precedes package.
-* `subpackages` - applies to all of the rules in the package and all subpackages (recursively)
+* `rule` - applies to the individual rule statement (within the same file). Default, when metadata block precedes rule.
+* `document` - applies to all of the rules with the same name in the same package (across multiple files)
+* `package` - applies to all of the rules in the package (within the same file). Default, when metadata block precedes package.
+* `subpackages` - applies to all of the rules in the package and all subpackages (recursively, across multiple files)
+
+Since the `document` scope annotation applies to all rules with the same name in the same package 
+and the `subpackages` scope annotation applies to all packages with a matching path, metadata blocks with 
+these scopes are applied over all files with applicable package- and rule paths. 
+As there is no ordering across files in the same package, the `document` and `subpackages` scope annotations 
+can only be specified **once** per path. 
+The `document` scope annotation can be applied to any rule in the set (i.e., ordering does not matter.)
 
-In case of overlap, schema annotations override each other as follows:
-
-```
-rule overrides document
-document overrides package
-package overrides subpackages
-```
-
-The following sections explain how the different scopes work.
-
-#### Rule and Document Scopes
-
-```
-# METADATA
-# scope: rule
-# schemas:
-#   - input: schema.input
-#   - data.acl: schema["acl-schema"]
-allow {
-    access := data.acl["alice"]
-    access[_] == input.operation
-}
-
-allow {
-    access := data.acl["bob"]
-    access[_] == input.operation
-}
-```
-
-In the example above, the second rule does not include an annotation, so type
-checking of the second rule would not take schemas into account. To enable type
-checking on the second (or other rules in the same file) we could specify the
-annotation multiple times:
-
-```
-# METADATA
-# scope: rule
-# schemas:
-#   - input: schema.input
-#   - data.acl: schema["acl-schema"]
-allow {
-    access := data.acl["alice"]
-    access[_] == input.operation
-}
-
-# METADATA
-# scope: rule
-# schemas:
-#   - input: schema.input
-#   - data.acl: schema["acl-schema"]
-allow {
-    access := data.acl["bob"]
-    access[_] == input.operation
-}
-```
-
-This is obviously redundant and error-prone. To avoid this problem, we can
-define the metadata block once on a rule with scope `document`:
+#### Example
 
-```
+```rego
```

Comment:
I think this should be a live-block (maybe a read_only one, but still), for consistent formatting and handling of Rego snippets on the website.

Reply:
I'll look into that 👍 .